jest.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('@ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/richardvanbergen/payload-plugin-cloud-storage/pullRequest?prid=7829468)
```suggestion
  testEnvironment: "node",
```